parachain/primitives/beacon/src/lib.rs

```diff
@@ -125,7 +125,6 @@ impl<'de> Deserialize<'de> for PublicKey {
 
 #[derive(Default, Encode, Decode, TypeInfo, MaxEncodedLen)]
 pub struct ExecutionHeaderState {
-	pub beacon_block_root: H256,
```

Comment:
@claravanstaden  Seems this field is not in use yet?  IMHO maybe some redundancy with `beacon_slot`, we can still retrieve from beacon-api later if nessessary. For saving runtime storage I remove it but correct me if I'm wrong.